app/organizations.js

```diff
@@ -1,33 +0,0 @@
-export default [
-  {
-    name: 'Discourse',
```

Comment:
It should be possible to keep the current organization.

Reply:
The implementation of organization will be replaced in the task 
https://gitlab.com/coala/GSoC/GSoC-2018/issues/312
using 
https://gitlab.com/coala/GSoC/GSoC-2018/issues/350

Reply:
that is for adding more.
Add these ones as part of this PR.
Otherwise this PR is a regression.